src/theory/sets/solver_state.h

```diff
@@ -204,6 +209,9 @@ class SolverState
   /** debug print set */
   void debugPrintSet(Node s, const char* c) const;
 
+  /** get the constant true */
+  const Node& getTrue() const { return d_true; }
```

Comment:
Can we remove this?  If you want to use true somewhere, either recreate it on the spot or cache it local to the class that is using it.

Reply:
Done.